### ISSUE TITLE
AMD Radeon 21.6.x drivers still incompatible with TRM 0.8.4 radeon vii

### ISSUE THREAD

**Issue by Cake-Chaser** [todxx/teamredminer#384]
Error: Failed to initialize device idx 0 (-13)
still works with 21.5.2

**Comment by echodreamz** [comment]
Yeah, the project doesnt seem overly active, not sure we will see a fix anytime soon.

**Comment by Kerney666** [comment]
We haven't been checking into github enough lately, support is primarily based on our discord server. 

Will try to boot my VIIs into win and handle this for the next release (0.8.6), didn't make it into 0.8.5 unfortunately. 99.9% of all VII users are on linux to be able to run our C-mode, partially the reason why this haven't been prioritized higher (yet).

**Comment by 9dagothur9** [comment]
Truning off HBCC did the thing. It works now. I wish I've tried it earlier. Thank you Kerney666!
